pkgs/top-level/all-packages.nix

```diff
@@ -23190,6 +23190,8 @@ in
 
   dcm2niix = callPackage ../applications/science/biology/dcm2niix { };
 
+  deeptools = python3Packages.callPackage ../applications/science/biology/deeptools { };
```

Comment:
```suggestion
  deeptools = callPackage ../applications/science/biology/deeptools { python = python3; };
```

Reply:
thanks for helping out with this, it opened new opportunities to learn about `with`.